lnrpc/walletrpc/walletkit.proto

```diff
@@ -962,6 +970,11 @@ message FundPsbtRequest {
 
     // Whether unconfirmed outputs should be used as inputs for the transaction.
     bool spend_unconfirmed = 7;
+
+    //  The scope for the change (if any). If empty, BIP0086 will be used for
+    //  the default account, or the same scope as the selection scope
```

Comment:
maybe add a more detailed description what the selection scope is here?

Reply:
What do you mean exactly?

By this comment, I meant that we will use the same scope as the account provided in FundPsbt (if not default)

Do you prefer:

`If empty, BIP0086 will be used for the default account. Otherwise, the change scope will be the same as the account provided.`
?

Reply:
will add a word about what is the coin selection scope